### PR TITLE
Add environment and service commands

### PR DESCRIPTION
**Pull request by evoxmusic**
Feature to add environment and service commands to the CLI:

- qovery environment list
- qovery environment deploy
- qovery environment redeploy
- qovery environment stop
- qovery environment cancel
- qovery environment clone
- qovery environment delete

- qovery container list
- qovery container deploy
- qovery container redeploy
- qovery container stop
- qovery container delete

### CODE REVIEW

**Comment by evoxmusic** [comment]
I also added commands for `application` and made some code refactoring to support `Authorization: Token ...` header

File: cmd/database_stop.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	},
+}
+
+func init() {
+	databaseCmd.AddCommand(databaseStopCmd)
+	databaseStopCmd.Flags().StringVarP(&organizationName, "organization", "", "", "Organization Name")
+	databaseStopCmd.Flags().StringVarP(&projectName, "project", "", "", "Project Name")
+	databaseStopCmd.Flags().StringVarP(&environmentName, "environment", "", "", "Environment Name")
+	databaseStopCmd.Flags().StringVarP(&databaseName, "database", "n", "", "Database Name")
+	databaseStopCmd.Flags().StringVarP(&databaseCommitId, "commit-id", "c", "", "Database Commit ID")
```
Comment (mzottola) [inline cmd/database_stop.go:68]: Can be removed

File: cmd/database.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package cmd
+
+import (
+	"github.com/qovery/qovery-cli/utils"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+var databaseName string
+var databaseCommitId string
```
Comment (mzottola) [inline cmd/database.go:10]: Can be removed, we don't have commit id for databases ?

**Review by erebe — APPROVED** [review]

**Comment by evoxmusic** [comment]
Thx @mzottola 🙌🏽

**Pull request merged**